test/acceptance/tests/terminating-gateway/terminating_gateway_namespaces_test.go

```diff
@@ -211,7 +211,7 @@ func TestTerminatingGatewayNamespaceMirroring(t *testing.T) {
 			}
 
 			// Create the config entry for the terminating gateway
-			createTerminatingGatewayConfigEntry(t, consulClient, ctx.KubectlOptions(t).Namespace, testNamespace)
+			createTerminatingGatewayConfigEntry(t, consulClient, "", testNamespace)
```

Comment:
This is a bug because we're always running acceptance tests in the default namespace. On openshift, we don't but we need to create the config entry in the default namespace in consul.